metron-platform/metron-common/src/main/java/org/apache/metron/common/configuration/writer/SingleBatchConfigurationFacade.java

```diff
@@ -18,6 +18,8 @@
 
 package org.apache.metron.common.configuration.writer;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Map;
 
```

Comment:
Well, there's some weirdness in this stuff, and I didn't try to get rid of it lest I break something I didn't understand.  In this case (and other places), it appears that non-batched writers are essentially faked by using batched writers with a batchSize of "1".  Doesn't seem real efficient to me, but obviously decreases the overall quantity of code and complexity of testing.  Anyway, since I only did a trivial change to this file, I didn't feel obligated to try to explain it.